### ISSUE TITLE
Ohio Positives from 4/28 to 4/29

### ISSUE THREAD

**Issue by Abrahajc** [COVID19Tracking/issues#333]
Prior to, and including 4/28, the Positives for Ohio were taking the "Confirmed Cases" on Ohio's Dashboard.  See tweet from Gov Dewine here:  https://twitter.com/GovMikeDeWine/status/1255204105015955457

However, on 4/29, the Positives were taken from the 'Total Cases' and not 'Confirmed Cases' (see attached picture).  This made it look like a tremendous jump in cases for Ohio from 4/28 to 4/29.  To fix this, it either makes sense to change 4/29 Positives to 'Confirmed Cases' or to change every other date before 4/29 to 'Total Cases' for the sake of day-to-day consistency.

<img width="684" alt="Ohio 4-29 Dash" src="https://user-images.githubusercontent.com/64562068/80653969-44d0c600-8a49-11ea-9061-9e20fefa1b73.PNG">

**Comment by Abrahajc** [comment]
Need to add 4/30 to the list.  4/30 is showing "total cases" instead of "confirmed cases."  I think it should be "confirmed cases," which is 17,205 for two reasons:

1. To match all of the reporting before 4/29
2. The "Total Cases" were not all tested, some were assumed positive without testing.  Since this data set includes total tests, it makes sense that positives would be people that actually received the test.

No idea how to get this updated...

**Comment by erfgottlieb** [comment]
👀

**Comment by erfgottlieb** [comment]
The Positives for OH have been updated to reflect the 'confirmed and probable cases' from 4/10 to 4/28. Prior to 4/10, the state was only reporting confirmed cases, so the Positives for these earlier dates have not been changed. We've also kept a record of the Confirmed numbers from 4/28 and earlier.
![Screen Shot 2020-05-05 at 10 08 39 AM](https://user-images.githubusercontent.com/64613495/81080129-67863300-8ebe-11ea-943d-2a40e7b9c9ba.png)
![Screen Shot 2020-05-05 at 10 42 18 AM](https://user-images.githubusercontent.com/64613495/81080148-6b19ba00-8ebe-11ea-8eb1-ba93972ce1bb.png)
